allensdk/brain_observatory/nwb/nwb_api.py

```diff
@@ -70,7 +70,7 @@ def get_invalid_times(self) -> pd.DataFrame:
         if container:
             return container.to_dataframe()
         else:
-            raise ValueError('There are no invalid time intervals in this session!')
```

Comment:
I think this should return an empty dataframe in this case.

Reply:
@NileGraddis The line 73 below does this exactly.